glib/src/enums.rs

```diff
@@ -227,6 +254,12 @@ impl fmt::Debug for EnumValue {
 }
 
 impl EnumValue {
+    // rustdoc-stripper-ignore-next
+    /// Creates a new `EnumValue` containing `value`.
+    pub const fn new(value: gobject_ffi::GEnumValue) -> Self {
```

Comment:
I think this should at least be unsafe. You can create a `GEnumValue` with random integers for the pointers, and then safe usage of that would cause memory corruption :)

Reply:
Maybe we can make use of `glib::UnsafeFrom` here? Basically, this feels like it should get very similar treatment to how `BoxedInline` works

Reply:
Sounds good to me